.changeset/calm-apes-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/fs-detectors': major
+---
+
+`LocalFileSystemDetector#readdir()` now returns paths relative to the root dir, instead of absolute paths. This is to align with the usage of the detectors that are using the `DetectorFilesystem` interface.
```

Comment:
Does "root dir" here mean Root Directory setting or repository root (or something else)?

Reply:
Root of the "virtual filesystem"